### ISSUE TITLE
Please include code relevant to the included libraries in the zio start project

### ISSUE THREAD

**Issue by boggye** [kitlangton/zio-start#5]
Hi,

Just a suggestion - it would it be very useful for newbies like me if the zio start project includes code relevant for the included libraries.

Example:
- If I select quill - include a config file to configure db access and include some code that queries that database. I think you could go with postgresql by default.
- If I select zio-http - include the code that starts up the server and maybe handles some requests.
- If I select zip-json - include some code that converts to or from json

A side note, I really liked what softwaremill did with this: https://bootzooka.softwaremill.com/. I actually experimented with it last night and I was able to add some testing api that queried a postgresql database. It's pretty cool.

Maybe you can call it Tapiokka 😂 (as you can tell I am hungry right now).

Thanks!

**Comment by kitlangton** [comment]
Definitely! Thanks for opening this issue. I was thinking of doing something like this. I'll check out bootzooka as well!
